tests/test_args.cpp

```
@@ -0,0 +1,70 @@
+
+#include <vector>
+
+namespace {
+
+// parseArgs takes argv as the C runtime hands it over, so the literals have to
+// shed their constness to get there.
+Args parse(const std::vector<const char*>& args) {
+  std::vector<char*> argv;
+  argv.reserve(args.size());
+  for (const char* arg : args) {
+    argv.push_back(const_cast<char*>(arg));
```

Clang (14.0.6)

[error] do not use const_cast

(cppcoreguidelines-pro-type-const-cast,-warnings-as-errors)